src/kOS/Suffixed/AddonList.cs

```diff
@@ -0,0 +1,33 @@
+﻿using kOS.Safe.Encapsulation;
+using kOS.Safe.Encapsulation.Suffixes;
+
+namespace kOS.Suffixed
+{
+    public class AddonList : Structure
+    {
+        private readonly Addon kacAddon;
+        private readonly Addon rtAddon;
+        private readonly Addon agxAddon;
+
+        public AddonList()
+        {
+            kacAddon = new Addon("KAC");

```

Comment:
We are still blocked here, structures should not have variable suffixes and knowledge about addons shouldnt leak out of the specific addon's namespace unless needed.

Please swap this out with a structure class for each addon. Thanks :)